### PR TITLE
[Wave Collect] [Xero/QBO] Update educational messages

### PR DESCRIPTION
**Pull request by mananjadhav**
### Fixed Issues

$ https://github.com/Expensify/App/issues/42456
PROPOSAL:

### Tests

Precondition:
- Have workspaces with accounting and QBO or Xero integration enabled.

**Test Steps: QBO**
1. Open the workspace with QBO integration enabled.
2. From the menu list, click on Categories.
3. Verify it shows the content for categories with QBO link.
4. Repeat the same for Tags and Taxes.

**Test Steps: Xero**
1. Open the workspace with Xero integration enabled.
2. From the menu list, click on Categories.
3. Verify it shows the content for categories with Xero link.
4. Repeat the same for Tags and Taxes. 

`These options can't be changed when [integrated with other systems]`

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/3069065/53df7b0d-569d-4a74-a3a9-ee0ae6707323

android-more-features-en

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/3069065/d2351fdb-91bd-4d11-bcad-211d16e11861

mweb-chrome-more-features-en

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/3069065/c68e8983-2568-4c6e-8fc9-89f25e5f40e4

ios-more-features-en

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/3069065/d92fae64-3c01-43a3-b250-93ca133bc25c

mweb-safari-more-features-en

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

**QBO Educational Messages EN**

web-qbo-categories-en

web-qbo-tags-en

web-qbo-taxes-en

**QBO Educational Messages ES**

web-qbo-categories-es

web-qbo-tags-es

web-qbo-taxes-es

**Xero Educational Messages EN**

web-xero-categories-en

web-xero-tags-en

web-xero-taxes-en

**Xero Educational Messages ES**

web-xero-categories-es

web-xero-tags-es

web-xero-taxes-es

**No Integration**

web-no-connection-categories-en

web-no-connection-tags-en

web-no-connection-taxes-en

**More Features**

web-more-features-en

web-more-features-es

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/3069065/8bc74f03-b337-4d37-94d0-2ba91e5f6939

desktop-more-features-en

</details>

### CODE REVIEW

**Comment by mananjadhav** [comment]
@rushatgabhane @hungvu193 @lakchote  This is ready for review.

**Comment by mananjadhav** [comment]
@dukenv0307 Please ignore this PR.

**Comment by mananjadhav** [comment]
Tagging @Expensify/design and @trjExpensify to verify the content and the styling.

**Comment by trjExpensify** [comment]
One more for the `Accounting` feature @mananjadhav, coming from [here](https://expensify.slack.com/archives/C036QM0SLJK/p1716412094186029?thread_ts=1716411028.782029&cid=C036QM0SLJK).

image

@Expensify/design when you review this PR, let us know if you want the spacing to be different. I think it looks good to me though. 👍 CC: @kevinksullivan

**Comment by trjExpensify** [comment]
I think we're missing a period at the end of each of these sentences: 

image

Compare that to the "default" when not connected to accounting for example:

image

**Comment by dubielzyk-expensify** [comment]
The Settings button here should span the whole width. It looks 50% here:
CleanShot 2024-05-23 at 13 00 32@2x

> @Expensify/design when you review this PR, let us know if you want the spacing to be different. I think it looks good to me though. 👍 CC: @kevinksullivan

Spacing wise I think it looks good, but I added an alternative way of doing this in Slack, but feel free to follow that up there

**Comment by shawnborton** [comment]
Good catch on the button width. Could have sworn we fixed that previously, so maybe we need a fresh pull of main or something?

**Comment by trjExpensify** [comment]
>Good catch on the button width. Could have sworn we fixed that previously, so maybe we need a fresh pull of main or something?

Yeah, @s77rt didn't you fix those? 🤔

**Comment by trjExpensify** [comment]
@mananjadhav for the messages on the `More features` page, looks like we're going to change direction and not add those messages in-line anymore, but rather go with an approach whereby: 

- Tap/clicking on a locked toggle shows an alert modal (bottom drawer on mobile/centered modal on web)
- The confirmation button takes you to the `Accounting` page to manage the integration settings

We're discussing and finalising copy [here](https://expensify.slack.com/archives/C036QM0SLJK/p1716459636970819?thread_ts=1716411028.782029&cid=C036QM0SLJK) for context. If you'd prefer to revert the `More features` portion out of this PR and follow-up with it separately, we can do that too. 👍

**Comment by s77rt** [comment]
The button width should be fixed already.  Merging `main` should fix this.

**Comment by trjExpensify** [comment]
> @mananjadhav for the messages on the `More features` page, looks like we're going to change direction and not add those messages in-line anymore, but rather go with an approach whereby:
> 
> * Tap/clicking on a locked toggle shows an alert modal (bottom drawer on mobile/centered modal on web)
> * The confirmation button takes you to the `Accounting` page to manage the integration settings
> 
> We're discussing and finalising copy [here](https://expensify.slack.com/archives/C036QM0SLJK/p1716459636970819?thread_ts=1716411028.782029&cid=C036QM0SLJK) for context. If you'd prefer to revert the `More features` portion out of this PR and follow-up with it separately, we can do that too. 👍

Modal copy finalised for you, Manan: 

image

**Comment by mananjadhav** [comment]
@trjExpensify @lakchote I've added the period. With the recent sync I think Settings button is also set.

https://github.com/Expensify/App/assets/3069065/8202372a-77cd-4412-b23e-b7e60c48d369

1. I reverted the More Features section. I can raise a separate PR for that.
2. When there are no Taxes in the QBO, the header text doesn't show in the small screen width.  I am not sure if this is the expected behavior, but unrelated to this PR.

**Comment by dubielzyk-expensify** [comment]
> Generally we don't allow on click handlers for disabled state. Do we want to override the behavior?

My take is yes on this one to achieve the pattern we want, but keen to hear what others think

**Comment by rushatgabhane** [comment]
hmmm that could get complicated. what if we don't make it disabled? just show the lock symbol in button that's clickable

**Comment by rushatgabhane** [comment]
and clicking the locked button would show the modal

**Comment by shawnborton** [comment]
> My take is yes on this one to achieve the pattern we want, but keen to hear what others think

That's my understanding too

> what if we don't make it disabled?

I agree with this, it shouldn't "feel" disabled but clicking it won't change the toggle, it will just launch the modal.

**Review by lakchote — APPROVED** [review]

**Comment by trjExpensify** [comment]
>If we're okay to keep it on(enabled) and just mimic the lock icon, then I'll implement that. 

Just to confirm on this comment, there are cases where the toggle is locked and off or locked and on. We don't want it to always be on as a result of this change to show a modal when clicked. We aren't doing that, right?

**Comment by mananjadhav** [comment]
> > If we're okay to keep it on(enabled) and just mimic the lock icon, then I'll implement that.
> 
> Just to confirm on this comment, there are cases where the toggle is locked and off or locked and on. We don't want it to always be on as a result of this change to show a modal when clicked. We aren't doing that, right?

Let me explain and see if it clarifies your question. The switch if `disabled`, will be **off and locked**. But for us to show modal, we shall have it `On` and show the `lock icon`. Click on it shows the modal but doesn't have any other side effects.

**Comment by trjExpensify** [comment]
>The switch if disabled, will be off and locked. But for us to show modal, we shall have it On and show the lock icon. 

Yeah, that doesn't sound like what we want. The "locked" state can apply to both the disabled and enabled state of the toggle. Therefore, the modal should show when clicked in both of those states.

**Comment by mananjadhav** [comment]
@trjExpensify does this video cover your concern?


https://github.com/Expensify/App/assets/3069065/27233083-2e3b-4a45-94f6-11f362aa0ac3

**Comment by rushatgabhane** [comment]
@mananjadhav does it work on "on" state of button too?

**Comment by rushatgabhane** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

**Comment by rushatgabhane** [comment]
https://github.com/Expensify/App/assets/29673073/f3607d90-2290-4b6b-a5c8-57e06c59d11b

https://github.com/Expensify/App/assets/29673073/cff165c0-e4c7-4f38-82fe-55ac97810fb2

**Comment by s77rt** [comment]
@mananjadhav 
> When there are no Taxes in the QBO, the header text doesn't show in the small screen width. I am not sure if this is the expected behavior, but unrelated to this PR

I think this is a bug (inconsistency). Can you move the getHeaderText call outside `<SelectionList />`

Can you also remove the "Test Steps: More Features" test since it's no longer relevant

**Review by s77rt — CHANGES_REQUESTED** [review]

File: src/languages/es.ts [hunk 1]
```diff
@@ -2189,6 +2189,10 @@ export default {
             organizeSection: {
                 title: 'Organizar',
                 subtitle: 'Agrupa y analiza el gasto, registra cada impuesto pagado.',
+                footer: {
+                    text: 'Estas opciones no se pueden cambiar cuando',
+                    link: 'integrado con otros sistemas',
+                },
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/languages/es.ts]: Done.

File: src/pages/workspace/tags/WorkspaceTagsPage.tsx [hunk 2]
```diff
@@ -320,7 +320,7 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
-                {!isSmallScreenWidth && getHeaderText()}
+                {!isSmallScreenWidth || (tagList.length === 0 && !isLoading) && getHeaderText()}
```
Comment (s77rt) [inline src/pages/workspace/tags/WorkspaceTagsPage.tsx:323]: Can you do same for the categories too?

File: src/pages/workspace/taxes/WorkspaceTaxesPage.tsx [hunk 3]
```diff
@@ -47,12 +49,16 @@ function WorkspaceTaxesPage({
...
     const {translate} = useLocalize();
+    const {environmentURL} = useEnvironment();
     const [selectedTaxesIDs, setSelectedTaxesIDs] = useState<string[]>([]);
     const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false);
     const defaultExternalID = policy?.taxRates?.defaultExternalID;
     const foreignTaxDefault = policy?.taxRates?.foreignTaxDefault;
     const isFocused = useIsFocused();
 
+    const isConnectedToAccounting = Object.keys(policy?.connections ?? {}).length > 0;
+    const isConnectedToQbo = policy?.connections?.quickbooksOnline;
```
Comment (hungvu193) [inline src/pages/workspace/taxes/WorkspaceTaxesPage.tsx:60]: Sorry for chiming in. But I think we should wrap it up with Boolean or use double negation operator here.
```suggestion
    const isConnectedToQbo = Boolean(policy?.connections?.quickbooksOnline);
```

File: src/pages/workspace/tags/WorkspaceTagsPage.tsx (hunk 2 above)
Comment (mananjadhav) [reply src/pages/workspace/tags/WorkspaceTagsPage.tsx]: The `isSmallScreenWidth` check is added later in the `SelectionScreen` `listHeaderComponent`. It seems it was added intentionally. So not sure about that.

File: src/pages/workspace/taxes/WorkspaceTaxesPage.tsx (hunk 3 above)
Comment (mananjadhav) [reply src/pages/workspace/taxes/WorkspaceTaxesPage.tsx]: Done.

**Review by hungvu193 — APPROVED** [review]
LGTM 🚀

File: src/pages/workspace/tags/WorkspaceTagsPage.tsx [hunk 4]
```diff
@@ -318,7 +320,7 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
-                {!isSmallScreenWidth && getHeaderText()}
+                {(!isSmallScreenWidth || (tagList.length === 0 && !isLoading)) && getHeaderText()}
```
Comment (s77rt) [inline src/pages/workspace/tags/WorkspaceTagsPage.tsx:323]:
```suggestion
                {(!isSmallScreenWidth || tagList.length === 0 || isLoading) && getHeaderText()}
```

**Review by s77rt — CHANGES_REQUESTED** [review]

File: src/pages/workspace/tags/WorkspaceTagsPage.tsx (hunk 4 above)
Comment (mananjadhav) [reply src/pages/workspace/tags/WorkspaceTagsPage.tsx]: Updated.

**Review by s77rt — APPROVED** [review]

**Comment by shawnborton** [comment]
Can we get some updated screenshots/videos please? Happy to do the final design review once I see them!

**Comment by lakchote** [comment]
> Can we get some updated screenshots/videos please? Happy to do the final design review once I see them!

@shawnborton I think the last ones from @s77rt are updated/the latest

**Comment by shawnborton** [comment]
@s77rt can you please update them to actually show the educational message flow? Or am I missing something perhaps?

**Comment by hungvu193** [comment]
Here you are @shawnborton 
For some reason, I can't connect to QBO at the moment.

https://github.com/Expensify/App/assets/16502320/3c3474dc-e3c2-449e-bb65-21ff5cc9e84a

**Comment by shawnborton** [comment]
Cool, that looks good. Are we still doing the modal pattern as well, [like this](https://github.com/Expensify/App/pull/42487#issuecomment-2127707089)?

**Comment by hungvu193** [comment]
> Cool, that looks good. Are we still doing the modal pattern as well, [like this](https://github.com/Expensify/App/pull/42487#issuecomment-2127707089)?

Yeah, we are. It was handled separately in this [PR](https://github.com/Expensify/App/pull/42662).

**Comment by shawnborton** [comment]
Ah okay, apologies for the confusion. Consider this approved from a design-perspective then!

**Comment by hungvu193** [comment]
All yours @lakchote

**Review by lakchote — APPROVED** [review]

**Pull request merged**